src/shared/protocol/game_builders.c

```
@@ -119,6 +121,26 @@ int bc_build_obj_not_found(u8 *buf, int buf_size, i32 object_id)
     return (int)b.pos;
 }
 
+int bc_build_enter_set(u8 *buf, int buf_size, i32 object_id, const char *set_name)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'bc_build_enter_set' of similar type are easily swapped by mistake

(bugprone-easily-swappable-parameters)